internal/core/data/router.go

```diff
@@ -1133,352 +1134,6 @@ func readingByValueDescriptorAndDeviceHandler(w http.ResponseWriter, r *http.Req
 	pkg.Encode(readings, w, LoggingClient)
 }
 
-// Value Descriptors
```

Comment:
See my other comment  regarding moving the REST handlers to another file.

Reply:
Disregard this comment. These changes have been reverted as per @tsconn23 's comments.